CMakeLists.txt

```diff
@@ -233,6 +233,9 @@ if("${CMAKE_CXX_COMPILER_ID}" MATCHES "MSVC")
 
 	# Default stack size is 1MB - increase to better match other platforms
 	set(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} /STACK:8000000")
+
+	# Enable multithreaded compilation by default
+	add_compile_options("/MP")
```

Comment:
We pass this in the CI workflow:
https://github.com/Warzone2100/warzone2100/blob/79232da5e9c2070e4b17895b00ecfdbbd0c9d9af/.github/workflows/CI_windows.yml#L510

I’m not sure if it’s ideal to add it here in the base CMake as well, as this seems more like an option that should be configured based on the environment and/or by whomever is compiling?

Reply:
Good idea. I'll change it to be a cmake option (ON by default), so that this can be disabled on demand.